src/main/kotlin/no/nav/familie/ef/sak/vedtak/historikk/AndelHistorikkBeregner.kt

```diff
@@ -91,6 +93,7 @@ object AndelHistorikkBeregner {
             val aktivitet = if (vedtaksperiode is VedtakshistorikkperiodeOvergangsstønad) vedtaksperiode.aktivitet else null
             val periodeType = if (vedtaksperiode is VedtakshistorikkperiodeOvergangsstønad) vedtaksperiode.periodeType else null
             val barnetilsyn = if (vedtaksperiode is VedtakshistorikkperiodeBarnetilsyn) vedtaksperiode else null
+            val sanksjon = if(vedtaksperiode is Sanksjonsperiode) vedtaksperiode else null
             val behandling = behandlingerPåId.getValue(it.behandlingId)
 
             AndelHistorikkDto(
```

Comment:
Det er mulig vi burde endre om AndelHistorikkDto til sealed class, spesielt når vi skal få med aktivitet/periodeType for barnetilsyn

Reply:
Ja - blir litt mye i denne nå. Ikke superfan av arv, men hvis det blir ryddigere er det kanskje ikke så dumt. 
  
Sealed, eller i større grad gjøre om AndelHistorikkDto til en aggregat med val os: OSAndelHistorikkDto mm.
 + validering?

Reply:
Med `Sealed class` tenkte jeg at en for hver stønadstype

Reply:
Hvilken typ av validering tenker du på? Nå er det bare flere elementer som er nullable, uten noen form av validering. 
I `VedtakHistorikkBeregner` er sealed class brukt for å separere på perioder for OS og BT